custom/enikshay/tests/test_adherence_updates.py

```diff
@@ -671,3 +671,53 @@ def test_update_by_person(self):
             {key: episode.dynamic_case_properties()[key] for key in expected_update},
             {key: str(val) for key, val in expected_update.iteritems()}  # convert values to strings
         )
+
+    def test_adherence_score_start_date_month(self):
+        # If the start date is more than a month ago, calculate the last month's scores
+        self.case_updater.date_today_in_india = datetime.date(2016, 1, 31)
+        self.assert_update(
```

Comment:
I know you're not introducing the function here, but these tests would be way more readable if `assert_update()` used kwargs instead of this massive nested tuple. e.g.
```python
def assert_update(
    self,
    purge_date=None, 
    adherence_schedule_date_start=None,
    adherence_schedule_id=None
    adherence_cases=None,
)
```